homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -238,13 +362,34 @@ async def async_setup_entry(hass, config_entry, async_add_entities):
         elif model in MODELS_HUMIDIFIER_MIIO:
             device = hass.data[DOMAIN][config_entry.entry_id][KEY_DEVICE]
```

Comment:
Assignment of device is all the same perhaps we can do the check for the condition in the else statement first.
Just get the value for `KEY_DEVICE` if key if it is available (perhaps use get).

Background:
`AIR_MONITOR_PLATFORMS` (`MODELS_AIR_MONITOR`) (```flow_type == CONF_DEVICE```) and `GATEWAY_PLATFORMS` (```flow_type == CONF_GATEWAY```)  are initializing then sensor platform too.